packages/grafana-data/src/types/query.ts

```diff
@@ -51,6 +53,11 @@ export interface DataQuery {
    * For non mixed scenarios this is undefined.
    */
   datasource?: DataSourceRef | null;
+
+  /**
+   * Specify the order in which the returned logs should be displayed initially
+   */
+  defaultLogsSortOrder?: LogsSortOrder;
```

Comment:
Can't we just get this based on direction? It seems like the same information as sent in `direction`. And then based on `direction` change the order.

Reply:
The problem is that `direction` is a Loki-only thing meaning we would have to do a type-cast to a plugin type inside a core component. I had it originally like that but it seemed wrong. However I am happy to do it if you think its alright.